.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<- Remember to label this issue accordly (bug, feature, help wanted, etc). And remove any information that does not fit the issue you are submitting.  -> 
```

Comment:
People who are not collaborators or in the OONI team cannot actually set labels (to set labels on github you need write permissions to the repo 🤦‍♂️ 

Maybe we should drop this sentence to avoid confusion.

Reply:
Yes. Forgot about checking about the labels.